### ISSUE TITLE
Pot generation is returning unexpected results

### ISSUE THREAD

**Issue by mhsdef** [Automattic/gp-localci-client#2]
We need to re-examine our pot generation logic as in spot checks the new strings pot doesn't match what I'm expecting. For example:

<img width="1010" alt="screen shot 2016-12-30 at 3 29 30 pm" src="https://cloud.githubusercontent.com/assets/2006764/21572219/cafb868e-cea4-11e6-99ca-f258fb5277ec.png">

We generated a new strings pot containing just two new strings. I manually checked all five shown in that PR and it should have been _four_ new--all but "Share post". Our master pot generation ran against `c37bfbf185d409687621eabca217e0954bb5782c` which is not what we want. We wanted master to be `ac2f4f20343396c4d0bbfaf7b105c069ec190f2b`.

tl;dr: https://github.com/Automattic/gp-localci-client/blob/master/generate-new-strings-pot.sh#L53 looks broken. We may way to just go a whole new way with the guts of the pot generation logic to satisfy this and #1.
